terraform/environments/digital-prison-reporting/athena_federated_queries.tf

```diff
@@ -0,0 +1,32 @@
+locals {
+  # TODO Parameterise NOMIS IP
+  #  Dev or Test for now
+  nomis_ip_address        = (local.environment == "dev") ? "10.26.24.29" : "10.26.12.239"
+  connection_string_nomis = "oracle://jdbc:oracle:thin:$${${aws_secretsmanager_secret.nomis_athena_federated.name}}@${local.nomis_ip_address}:1521:CNOMT3"
+}
+
+module "athena_federated_query_connector_oracle" {
+  source = "./modules/athena_federated_query_connectors/oracle"
+
+  nomis_cidr                   = "${local.nomis_ip_address}/32"
+  spill_bucket_name            = module.s3_working_bucket.bucket_id
+  connector_jar_bucket_name    = module.s3_artifacts_store.bucket_id
+  connector_jar_bucket_key     = "third-party/athena-connectors/athena-oracle-2024.18.2.jar"
```

Comment:
@harichintala1 So far I have manually uploaded this jar. Do you think we should instead put it in version control in the repo (I think we do this for other lambdas)? I think it would be updated very rarely.

Reply:
We said we'd manually upload the jar for now and discuss raising a tech debt ticket to automate uploading of any 3rd party jars